core/services/wifi/frontend/index.html

```diff
@@ -0,0 +1,312 @@
+          this.closeErrorAlert()
+          this.alert_box = Metro.infobox.create(`
+            <div id='alert_box'>
+              <h3>Error</h3>
+              <p>${message}</p>
+            </div>
+            `, 'alert')
+        },
+        openConnectionDialog (network) {
+          this.selected_network = network
+          Metro.dialog.open('#connection_dialog')
+        },
+        openDisconnectionDialog () {
+          Metro.dialog.open('#disconnection_dialog')
+        },
+        signalStrengthClass (network) {
+
+          /*eslint-disable */
+          // | Signal Strength | TL;DR     |  Description                                                                                                                               |
+          // |-----------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------|
+          // | -30 dBm         | Amazing   | Max achievable signal strength. The client can only be a few feet from the AP to achieve this. Not typical or desirable in the real world. |
+          // | -67 dBm         | Very Good | Minimum signal strength for applications that require very reliable, timely delivery of data packets.                                      |
+          // | -70 dBm         | Okay      | Minimum signal strength for reliable packet delivery.                                                                                      |
+          // | -80 dBm         | Not Good  | Minimum signal strength for basic connectivity. Packet delivery may be unreliable.                                                         |
+          // | -90 dBm         | Unusable  | Approaching or drowning in the noise floor. Any functionality is highly unlikely.                                                           |
+          // Reference: metageek.com/training/resources/wifi-signal-strength-basics.html
+          /*eslint-enable */
+
+          if (network.signallevel >= -67) return 'mif-wifi-full'
+          if (network.signallevel >= -70) return 'mif-wifi-mid'
+          if (network.signallevel >= -80) return 'mif-wifi-low'
+          return ''
+        },
+        networkProtectionClass (network) {
+          if (!network.flags) return 'mif-question'
+          return network.flags.includes('WPA') ? 'mif-lock' : ''
+        },
+        sleep (ms) {
+          return new Promise((resolve) => setTimeout(resolve, ms))
+        },
+      },
+    })
+    </script>
+
+  <style>
+    .connected-network {
+      background-color: #00aff0
+    }
+
+    .unconnected {
+      background-color: #64a0b6
+    }
+
+    .connected-network:hover {
+      cursor: pointer;
+    }
+    
+    .available-network {
+      background-color: #f8f8f8;
+    }
+    
+    .available-network:hover {
+      cursor: pointer;
+      background-color: #c5c5c5;
+    }
+
+    body {
+        background-color: #ECF0F5;
```

Comment:
indentation is not consistent

Reply:
eslint is supposed to detect that ?

Reply:
I really assumed eslint was fixing that stuff. Will check next time. Tks.